src/window.ts

```diff
@@ -424,8 +495,6 @@ export class SnapAssistWindowManager {
             y = (r2.y + r2.height) - r1.height;
         }
 
-        return (r1.x !== x || r1.y !== y)
-            ? new Rectangle(x, y, r1.width, r1.height)
-            : undefined;
+        return new Rectangle(x, y, r1.width, r1.height);
```

Comment:
This was originally there in an attempt to eliminate having to move a window if it is already snapped but made it too difficult to maintain showing the target group hint (eg: the transparency on target windows) when you were already snapped and were dragging the window along the edge.